iroh/src/magicsock/metrics.rs

```diff
@@ -20,6 +20,7 @@ pub struct Metrics {
     // Data packets (non-disco)
     pub send_data: Counter,
     pub send_data_network_down: Counter,
+    pub send_pending: Counter,
```

Comment:
I know existing metric names do not do this, but could we follow https://prometheus.io/docs/practices/naming/ for new metric names?

I think this would be `send_would_block_total`, the prefix is already done by the framework here.

/cc @Arqu

Reply:
The current setup automatically adds _total to all counters, unless you define a unit on the counter which we currently don't do anywhere. So technically we are using the correct metrics naming convention, just not utilizing it fully.